bmwqemu.pm

```diff
@@ -51,7 +51,7 @@ our @ovmf_locations = (
 our %vars;
 tie %vars, 'bmwqemu::tiedvars', %vars;
 
-sub result_dir { 'testresults' }
+sub result_dir() { 'testresults' }
```

Comment:
```suggestion
sub result_dir () { 'testresults' }
```
I suppose we normally use a space here, don't we? So I'd use it consistently in places where signatures are used.

Reply:
you can teach perltidy :)

Reply:
With

```
git grep 'sub [a-z_A-Z0-9]\+()' | wc -l ; git grep 'sub [a-z_A-Z0-9]\+ ()' | wc -l
```

I found that there are 15 references of subs without separating space before `()` and 27 with a space. I suggest to allow both styles for now and try to make that consistent in a separate step. I prepared a PR for that: https://github.com/os-autoinst/os-autoinst/pull/2021

Reply:
I found 15 and 27 a bit too less :)
```
git grep 'sub [a-z_A-Z0-9]\+(' | wc -l ; git grep 'sub [a-z_A-Z0-9]\+ (' | wc -l
19
665
```

Reply:
Let's handle that in #2021 . perltidy already adds the space for any non-zero parameter signature list.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/2021 merged, rebase PR. Now we have consisent `sub <name> () {…`